### PR TITLE
feat(decorator): added optional title prop

### LINKED ISSUE #322: Request: `Decorator` to accept `title` property
### Details
Have `Decorator` component to accept a `title` prop to be used when texts are truncated

### Screenshots
Screen Shot 2020-01-22 at 1 53 35 PM

### Proposal
`Decorator` component to accept a `title` prop

### PR DESCRIPTION
**Pull request by i0r1**
## Affected issues

- Resolves #322 

## Proposed changes

- added optional title prop

### CODE REVIEW

**Comment by SimonFinney** [comment]
@jendowns I'd be up for that - Perhaps we can create a follow-up to return to that to prevent this from being blocking for the application team?

File: src/components/DataDecorator/Decorator/Decorator.js [hunk 1]
```diff
@@ -127,6 +129,9 @@ Decorator.propTypes = {
     return null;
   },
 
+  /** @type {string} The title of data. */
+  title: PropTypes.string,
```
Comment (jendowns) [inline src/components/DataDecorator/Decorator/Decorator.js]: Can you also add this prop to the `DataDecorator` & pass it through to the inner `Decorator`?

Comment (i0r1) [reply src/components/DataDecorator/Decorator/Decorator.js]: Added

**Comment by SimonFinney** [comment]
Hi @i0r1 ! Can you confirm if you're an IBMer? If so, please reach out on Slack and to organise receiving a badge for your contribution

**Pull request merged**